WalletWasabi/Blockchain/Transactions/TransactionFactory.cs

```diff
@@ -356,6 +359,24 @@ private void UpdatePSBTInfo(PSBT psbt, SmartCoin[] spentCoins, HdPubKey changeHd
 					psbt.AddKeyPath(changeHdPubKey.PubKey, rootKeyPath, changeHdPubKey.P2wpkhScript);
 				}
 			}
+
+			if (Store is null)
```

Comment:
I would prefer this in the other way, we don't have this kind of circuit breakers.

Reply:
Im not sure what you meant... should i invert the if statement instead?

Reply:
In the practice, this is for hacking the functionality for tests, am I right? In the functional code, BitcoinStore should not be null. So that should be protected with guard and in the test use some mock object.